docs/source/docs/level-up/basic_skills/01_project_manipulation.rst

```diff
@@ -146,11 +107,13 @@ Print project information
 
   .. tab-item:: ProjectCLI
 
-    Print an overall information of the project.
+    We can print an overall information of the project.
 
     .. code-block:: bash
 
-      datum project pinfo [-h] [-p PROJECT_DIR] [revision]
+      datum project info
```

Comment:
is this correct? `pinfo`

Reply:
- https://github.com/openvinotoolkit/datumaro/pull/952 fixed it to `datum project info`.